test/unit/units/user_test.py

```diff
@@ -70,17 +77,34 @@ def side_effect_user_exists(user):
             ]
             assert mock_open.call_args_list == [
                 call('/home/hanauser/.ssh/authorized_keys', 'a'),
+                call('/root/.sshauthorized_keys', 'a'),
                 call('/etc/sudoers', 'a')
             ]
             assert file_handle.write.call_args_list == [
+                call('\n'),
+                call('ssh-rsa foo'),
                 call('\n'),
                 call('ssh-rsa foo'),
                 call('\n'),
                 call('%admin ALL=(ALL) NOPASSWD: ALL')
             ]
             assert mock_chmod.call_args_list == [
                 call('/home/hanauser/.ssh/', 0o700),
-                call('/home/hanauser/.ssh/authorized_keys', 0o600)
+                call('/home/hanauser/.ssh/authorized_keys', 0o600),
+                call('/root/.ssh', 0o700),
+                call('/root/.sshauthorized_keys', 0o600)
```

Comment:
Here too? Or am I misunderstanding something?

Reply:
yes you are right, I saw it too when my test machine came back with an error and was about to force push the part I overlooked, but you were faster :) Thanks